torchtext/modules/multiheadattention.py

```diff
@@ -3,14 +3,18 @@
 
 
 class MultiheadAttentionContainer(torch.nn.Module):
-    def __init__(self, nhead, in_proj_container, attention_layer, out_proj):
+    def __init__(self, nhead, in_proj_container, attention_layer, out_proj, batch_first=False):
         r""" A multi-head attention container
 
         Args:
             nhead: the number of heads in the multiheadattention model
             in_proj_container: A container of multi-head in-projection linear layers (a.k.a nn.Linear).
-            attention_layer: The attention layer.
+            attention_layer: The custom attention layer. The input sent from MHA container to the attention layer
```

Comment:
Does this also take care of broadcasting?

Reply:
The custom attention layer needs to take care of broadcasting. Updated the doc to reflect this.

Reply:
I'd then augment the shape to ```(..., seq, batch, feature)``` and explain what that means and also that it's optional, i.e. enough to only handle 3-dim.